tests/gold_tests/pluginTest/slice/slice_regex.test.py

```diff
@@ -126,7 +126,6 @@
 ts.Disk.records_config.update({
 #  'proxy.config.diags.debug.enabled': 1,
 #  'proxy.config.diags.debug.tags': 'slice',
-  'proxy.config.http.cache.http': 0,
   'proxy.config.http.wait_for_cache': 0,
```

Comment:
Should this line go away with the enable_cache=False?

Reply:
Removed.